Makefile

```diff
@@ -227,6 +227,7 @@ printf "$$rows" Portainer 'http://$(get_my_ip).nip.io:9000' admin adminadmin;\
 printf "$$rows" Redis 'http://$(get_my_ip).nip.io:18081';\
 printf "$$rows" 'Docker Registry' $${REGISTRY_URL} $${REGISTRY_USER} $${REGISTRY_PW};\
 printf "$$rows" "Dask Dashboard" "http://$(if $(IS_WSL2),$(get_my_ip),127.0.0.1).nip.io:8787";
+printf "$$rows" "Traefik Dashboard" "http://$(if $(IS_WSL2),$(get_my_ip),127.0.0.1).nip.io:8080/dashboard/";
```

Comment:
when i work remotely, I would like to have the ip, otherwise I have to be forwarding all ports :-)

Reply:
I think we should only use the `$(get_my_ip)`, we already did this above.

Reply:
@sanderegg  so why we don't?